packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -11,6 +11,8 @@ import 'package:flutter/painting.dart';
 import 'package:vector_math/vector_math_64.dart';
 
 import 'node.dart';
+import 'object.dart';
+import 'viewport.dart';
```

Comment:
Can you document this extended layer cake somewhere so people can refer to it? According to the layer cake in https://flutter.io/technical-overview/ this is all in the same layer.